### PR TITLE
New/npm package for angular components

### PR DESCRIPTION
**Pull request by proand**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,17 +1,58 @@
...
+  - [BEM](#bem)
+    - [BEM in a nutshell](#bem-in-a-nutshell)
+- [Documentation](#documentation)
+- [License](#license)
+
+<!-- /code_chunk_output -->
+
+## Git submodules
+
+There are two submoduls in this repo:
```
Comment (yleaxeman) [inline CONTRIBUTING.md:38]: There is one submodule in this repo

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,17 +1,58 @@
...
+
+- `./Fhi.Frontend.Style`, [Github repo Fhi.Frontend.Style](https://github.com/folkehelseinstituttet/Fhi.Frontend.Style)
+- `./projects/fhi-angular-components`, [Github repo Fhi.Frontend.AngularComponents](https://github.com/folkehelseinstituttet/Fhi.Frontend.AngularComponents)
```
Comment (yleaxeman) [inline CONTRIBUTING.md:41]: This repo will be deleted?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,17 +1,58 @@
...
 Requests for new features and bug reports keep the project moving forward.
 
 ### Before you submit an issue
 
-- Ensure you are running the [latest version](https://github.com/folkehelseinstituttet/Fhi.Frontend.Demo/releases) of Fhi.Frontend.Demo
-- **Search** the [issue list](https://github.com/folkehelseinstituttet/Fhi.Frontend.Demo/issues?utf8=✓&q=is%3Aissue) (including closed issues) to make sure it hasn't already been reported.
+- Ensure you are running the latest version of Fhi.Frontend.Demo and its submodules.
+- **Search** the issue lists (including closed issues) to make sure it hasn't already been reported.
+  - [Issue list Fhi.Frontend.Style](https://github.com/folkehelseinstituttet/Fhi.Frontend.Style/issues?utf8=✓&q=is%3Aissue)
+  - [Issue list Fhi.Frontend.AngularComponents](https://github.com/folkehelseinstituttet/Fhi.Frontend.AngularComponents/issues?utf8=✓&q=is%3Aissue)
```
Comment (yleaxeman) [inline CONTRIBUTING.md:54]: Angular module issues will be part of issues list in Fhi.Frontend.Demo, right?

File: README.md [hunk 4]
```diff
@@ -5,20 +5,11 @@ An Angular app with demos and documentation of CSS and Angular components with F
 
 ## Get started
 
->**NB!** If you clone the repository without the `--recurse-submodules` flag, the subdirectories named `Fhi.Frontend.Style` and `projects/fhi-frontend-angular-components` will be empty. If thats the case, you have to initialize the submodules and clone their content by running `git submodule update --init --recursive`
+>**NB!** If you clone the repository without the `--recurse-submodules` flag, the subdirectories `./Fhi.Frontend.Style` and `./projects/fhi-angular-components` will be empty. If thats the case, you have to initialize the submodules and clone their content by running `git submodule update --init --recursive`
```
Comment (yleaxeman) [inline README.md:8]: Is `./projects/fhi-angular-components` still a submodule?

File: projects/fhi-angular-components/README.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
+# FHI AngularComponents
+
+Contains frontend Angular components used by FHI, based on [Bootstrap widgets (@ng-bootstrap)](https://ng-bootstrap.github.io) and [Fhi.Frontend.Style (@folkehelseinstituttet/style)](https://www.npmjs.com/package/@folkehelseinstituttet/style)
+
+## Dependencies
+
+| FHI AngularComponents | Fhi.Frontend.Style | Bootstrap | NgBootstrap | Angular |
+| --------------------- | ------------------ | --------- | ----------- | ------- |
+| 0.0.1                 | 3.10.0             | 5         | 13          | 14      |
```
Comment (yleaxeman) [inline projects/fhi-angular-components/README.md:9]: Should we state Node version and npm version as well?

**Comment by proand** [comment]
Might still be some typos, but it's getting better and better 😄

File: README.md [hunk 6]
```diff
@@ -5,20 +5,11 @@ An Angular app with demos and documentation of CSS and Angular components with F
 
 ## Get started
 
->**NB!** If you clone the repository without the `--recurse-submodules` flag, the subdirectories named `Fhi.Frontend.Style` and `projects/fhi-frontend-angular-components` will be empty. If thats the case, you have to initialize the submodules and clone their content by running `git submodule update --init --recursive`
+>**NB!** If you clone the repository without the `--recurse-submodules` flag, the subdirectory `./Fhi.Frontend.Style`. If that's the case, you have to initialize the submodule and clone it's content by running `git submodule update --init --recursive`
```
Comment (yleaxeman) [inline README.md:8]: The first sentence is missing "will be empty" at the end?

**Review by yleaxeman — APPROVED** [review]

**Review by yleaxeman — APPROVED** [review]

**Pull request merged**